WordPress/Classes/ViewRelated/Stats/JetpackLoginViewController.swift

```diff
@@ -82,6 +89,10 @@ class JetpackLoginViewController: UIViewController {
         updateMessageAndButton()
     }
 
+    private func hideImageView() {
+        jetpackImage.isHidden = WPDeviceIdentification.isiPhone() && UIDevice.current.orientation.isLandscape
```

Comment:
I think rather than limiting this to iPhone / landscape, we could do like we do in `FancyAlertViewController` and test the vertical size class:

```
override open func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
        super.traitCollectionDidChange(previousTraitCollection)

        if traitCollection.containsTraits(in: UITraitCollection(verticalSizeClass: .compact)) {
            // hide image view
        } else {
            // show image view
        }
    }
```

Reply:
Thanks @frosty ! This is much better!